tutorials/getting-started-with-flow/01__how-to-use-vaadin-flow/content.adoc

```diff
@@ -33,12 +33,12 @@ https://github.com/vaadin/tutorial/tree/vaadin10+[download the final application
 === Installing the Development Tools 
 
 
-* *JDK 8*: The tutorial uses Java 8. Make sure that you have JDK 8 or later installed and that it is up to date.
-** Most Linux distributions can use package managers to install JDK 8.
-** Windows and Mac users need to download JDK 8 from http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle's Java SE Downloads].
+* *JDK*: The tutorial uses Java 8. Make sure that you have JDK 8 or later installed and that it is up to date.
+** Most Linux distributions can use package managers to install JDK.
+** Windows and Mac users need to download JDK from http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle's Java SE Downloads].
 
-* *IDE*: The tutorial uses Eclipse, but you can use any IDE. Make sure you have the latest version installed. 
-** To download Eclipse, select *Eclipse IDE for Java EE Developers* from the available packages at https://www.eclipse.org/downloads/packages/[eclipse.org].
+* *IDE*: The tutorial uses https://www.eclipse.org[Eclipse], but you can use other IDE that you already master. Make sure you have the latest version installed. 
```

Comment:
It is quite complex to find the correck pack navigating from here - https://www.eclipse.org. This - https://www.eclipse.org/downloads/packages/ - leads directly to the correct one

Reply:
The right corner has a download link. Maybe we should point to https://www.eclipse.org/downloads/ directly. If I have understood it correctly the https://www.eclipse.org/downloads/packages site is kind of legacy thing, now replaced with the installer app.